File: share-code/bai-tap-lap-trinh/cpp/30.cpp
#include <iostream>
using namespace std;
#include <string>

int main()
{
    int start = 4;
    int end = 5;
    string result = "";
    for (int i = start + 1; i < end; i+2) 
    {
       {result += i + " ";}
    }
    if(result.length() == 0) {result = "Không có";}
    cout << result << endl;
}

File: share-code/bai-tap-lap-trinh/python/loop/30.py
start = 4
end = 5

# Cách 1:
result = ""

for i in range(start + 1, end):
    if i % 2 == 0:
        result += str(i) + " "

result = result.rstrip() if result else "Không có"

# Cách 2:
# even_numbers = [str(i) for i in range(start + 1, end) if i % 2 == 0]

# result = " ".join(even_numbers) if even_numbers else "Không có"

print(result)